postgres/setup.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+
+set -e
+
+sed -i "/host all all all/d" "$PGDATA/pg_hba.conf"
+echo "host all $WUBLOADER_USER all md5" >> "$PGDATA/pg_hba.conf"
```

Comment:
add comment explaining what these two lines do? I can tell it's replacing a line in the config file, but what does that line mean?